pithos/application.py

```diff
@@ -85,6 +87,38 @@ def do_command_line(self, command_line):
         # http://stackoverflow.com/questions/1943747/python-logging-before-you-run-logging-basicconfig
         logging.root.handlers = []
 
+        # Show the Pithos log since last reboot and exit
+        if options.contains('last-logs'):
+            try:
+                from systemd import journal
+                from os.path import basename
+            except ImportError:
+                type(command_line).do_print_literal(command_line, 'Systemd Python module not found')
```

Comment:
On a closer look it really doesn't miss much anyway at the INFO level...Not really worth the trouble for a hand off.